app/webpack/components/authentication/sign_in.vue

```diff
@@ -324,6 +365,32 @@ export default {
             '_self'
           );
         });
+    },
+    handleFirebaseError(error) {
+      this.showInstructionalText = false;
+      this.showFailureText = true;
+      switch (error.code) {
+        case 'auth/unsupported-persistence-type':
+          this.failureText =
+            'There may be a problem with your cookies or storage that is stopping us from signing you in. Please check your settings.';
+          break;
+        case 'auth/network-request-failed':
+          this.failureText =
+            'It looks like there is an issue with your network or connection that is preventing us from signing you in. Please check your settings.';
+          break;
+        case 'auth/too-many-requests':
+          this.failureText =
+            'It looks like there have been a lot of sign in attempts from this computer in the past hour so we are blocking any more attempts for security reasons. Please try again later.';
+          break;
+        case 'auth/user-disabled':
+          this.failureText =
+            'The account associated with this email address has been disabled. Please contact us through our about page if you are not sure why.';
+          break;
+        case 'server/signin':
+          break;
```

Comment:
Recommend we set an error message like:

"Sorry, something went wrong when signing you in! We've been notified about this. Please try again later on".

… both in this `case` branch and in the `default` below.

(Nice one on not registering a Rollbar error from the client side here [when a server error has occurred], since you already register one from the server 👍)

Reply:
Done!